presto-memory/src/test/java/io/prestosql/plugin/memory/TestMemorySmoke.java

```diff
@@ -43,6 +44,10 @@
 public class TestMemorySmoke
         extends AbstractTestQueryFramework
 {
+    private static final long LINEITEM_COUNT = 60175;
```

Comment:
please add a test case where build side is large. see https://github.com/prestosql/presto/pull/4946, e.g:
```
SELECT * FROM orders a JOIN tpch.tiny.orders b ON a.orderkey = b.orderkey
```
The query should not be blocked

Reply:
Cherry-picked 0f349890a0 and added `testJoinLargeBuildSideNoDynamicFiltering`.